Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -19,16 +19,60 @@
 
 import WebKit
 
-public class AutofillUserScript: NSObject, UserScript {
+public protocol AutofillUserScriptDelegate: AnyObject {
+    func setSize(height: CGFloat, width: CGFloat)
+}
+public protocol ChildAutofillUserScriptDelegate: AnyObject {
+    func clickTriggered(clickPoint: NSPoint)
+}
+
+public protocol AutofillMessagingToChild {
+    var lastOpenHost: String? { get }
+    func messageSelectedCredential(_ data: [String: String], _ configType: String)
+    func close()
+}
+
+public struct InputDimensions {
```

Comment:
If you wanted I think you could use a [`CGRect`](https://developer.apple.com/documentation/coregraphics/cgrect) in place of this custom struct here, since it encodes origin & size already. Not really required though, this is easy to follow as is.

Reply:
Agreed thanks, fixed.